cards/src/main/resources/cards/custom/group0/other_death_knight/ungoro/minion_akarador.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Akarador",
+  "baseManaCost": 4,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 3,
+  "baseHp": 3,
+  "rarity": "LEGENDARY",
+  "description": "Battlecry: Summon the last friendly minion that died this game.",
```

Comment:
You might want to introduce a `FRIENDLY_LAST_MINION_DIED` target, simply, and store it in the environment. Also test this.